NSS-Likeable-Lemurs/BangazonSite
# Hc products ticket4

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/BangazonSite/issues/4)

## Description of Proposed Changes
- Edited HomeController

## Steps to Test

- Fire up the project - see that 20 products listed (most recent) on the home page whether logged in or not.
- Each project is linked to their detail page

Outline the steps to test

```sh
git fetch --all
git checkout <HC-Products-Ticket4>
Fire Up Project and test steps listed above
Check for comments
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* HomeController and HomeIndexView

## Mentions @username

@streatorward 
@kellycook301 
@stjepad
@ThraxHamilton 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
